src/socket.rs

```diff
@@ -996,6 +996,35 @@ impl Socket {
         }
     }
 
+    /// Get value for the `SO_PRIORITY` option on this socket.
+    ///
+    /// For more information about this option, see [`set_priority`].
+    ///
+    /// [`set_priority`]: Socket::set_priority
+    #[cfg(any(target_os = "linux", target_os = "android", target_os = "fuchsia"))]
```

Comment:
This needs the `all` feature if it's not available on Windows and macOS as well. See the `cfg` that uses `features = "all"` on some other methods.